modules/bonita-overview/pages/bonita-bpm-overview.adoc

```diff
@@ -1,6 +1,6 @@
 = Bonita components
 :page-aliases: ROOT:bonita-bpm-overview.adoc
-:description: Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
+:description: Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams.
```

Comment:
At the same time, can you fix the misspelling of `Morevoer` of the next sentence please :pray:

Reply:
Well.. later then...

Reply:
oops... I've seen this 2s too late... Merged.